libs/sm/cleanup.py

```diff
@@ -1075,14 +1082,16 @@ def _getCoalescedSizeData(self):
         blocksParent = self.parent.getVHDBlocks()
         numBlocks = Util.countBits(blocksChild, blocksParent)
         Util.log("Num combined blocks = %d" % numBlocks)
-        sizeData = numBlocks * vhdutil.VHD_BLOCK_SIZE
+        sizeData = numBlocks * self.block_size
```

Comment:
```suggestion
        sizeData = numBlocks * self.block_size()
```

Reply:
`self.block_size` should be treated as an attribute due to the use of the `@property` decorator. I don't think your suggestion will work in that case 🤔